test/pipelines/test_eval.py

```diff
@@ -420,7 +419,6 @@ def test_eval_data_split_passage(document_store, samples_path):
 ]
 
 
-@pytest.mark.integration
```

Comment:
I agree that this is neither an integration test nor a unit test. It's not run at the moment and removing the marker won't change that.

Reply:
That's a fair point, but might as well remove it since it isn't relevant.

Reply:
I disagree, I'd keep it marked and explicitly skip it with a clear reason why.

Adding new unmarked tests just makes it harder to maintain them.

Reply:
I would even prefer deletion to have it unmarked to be fair.

Reply:
Out of these options skipping with `@pytest.mark.skip(reason="...")` seems best to me. That's then also a better option for the above `test_add_eval_data` with ` # TODO This currently fails for Weaviate and Pinecone`. Improving these tests and then reactivating them is still a task. By deleting such tests it becomes less visible what remains untested. I'd prefer a cleaner code base too but if we don't have time to improve that test now let's leave it there as a skipped test to come back to.

Reply:
And I also agree with this sentiment 

> Improving these tests and then reactivating them is still a task. By deleting such tests it becomes less visible what remains untested.

Reply:
The `integration` mark is misleading in any case, most of those are just long running tests and at a certain point renamed to integration.

Leaving the mark `integration` but also marking it skipped will keep it visible in the list of tests when they're run. 
If we leave it unmarked it will still run if you don't specify a mark when running `pytest` and that's quite frustrating.

Also making it skipped explicitly makes it clear that's skipped on purpose instead by accident.

Reply:
@sjrl Sorry for any confusion caused by two different reviewer opinions here. Silvano and I quickly aligned offline now. We agree on that the test should be skipped with the following, yes:
```
@pytest.mark.skip(reason="Should be an end-to-end test")
@pytest.mark.integration
```